src/main/export.ts

```diff
@@ -33,14 +34,22 @@ const constructEDL: (title: string, transcription: Transcription) => string = (
   return output;
 };
 
-export const exportEDL = (project: Project) => {
+export const exportEDL = (
+  mainWindow: BrowserWindow | null,
+  project: Project
+) => {
   mkdir(project.savePath);
 
   if (project.transcription) {
     writeFileSync(
       join(project.savePath, `${project.name}.edl`),
       constructEDL(project.name, project.transcription)
     );
+
+    // Placeholder for BE - to be copied and removed
+    mainWindow?.webContents.send('export-progress-update', 0.5);
```

Comment:
This line & comment to be removed in a later PR. Probably by @itsMoxMox 

Included so that backend team can easily copy the functionality into the export logic so that export progress can be updated at intervals throughout.

Reply:
Thanks for the code nugget!